README.template.md

```diff
@@ -460,6 +460,16 @@ EMAIL=postmaster@example.com
 
 > N.B. This is required to property generate certificates automatically!
 
+## Using a Wildcard Certificate
+
+If your production Islandora site is using a wildcard certificate that is not provisioned by LetsEncrypt, your DNS entries for your subdomain `DOMAIN` may need to be at the same level as your wildcard's top level domain (TLD).
+
+For example if your wildcard cert is for `*.library.world.com` and your islandora site is at `islandora.library.world.com`, to have your `activemq`, `blazegraph`, `fcrepo`, and `solr` subdomains use the same TLS certificate, they will need to be children of `library.world.com`. To accomplish this, you can change the subdomain delimiter in your `.env` to use a dash (-) instead of a period (.).
```

Comment:
I'm confused reading this. If I used a dash, would I not get `activemq-library.world.com`? I think I must be misreading this, and I'm hoping we can clarify the wording.

Reply:
> If I used a dash, would I not get activemq-library.world.com

`DOMAIN` is set to the drupal site. In the example, `islandora.library.world.com`.

The fcrepo/activemq/etc. domains would then be `activemq-islandora.library.world.com`. These get set in the [docker-compose.yml](https://github.com/Islandora-Devops/isle-site-template/blob/afa3e82d7fec510c31f17c5788e024ce8c087cc9/docker-compose.yml#L675-L683)

`activemq-islandora.library.world.com` is a child domain of `library.world.com` so would be able to serve TLS connections from a wildcard cert with `*.library.world.com` (as would `islandora.library.world.com`)